kfdefs/overlays/moc/smaug/opf-trino/vpa/vpa-custom-resource.yaml

```diff
@@ -0,0 +1,24 @@
+---
+apiVersion: autoscaling.k8s.io/v1
+kind: VerticalPodAutoscaler
+metadata:
+  namespace: opf-trino
+  name: vpa-trino
+  labels:
+    app: vpa-trino
+spec:
+  resourcePolicy:
+    containerPolicies:
+      - containerName: trino-worker
+        maxAllowed:
+          cpu: 8000m
```

Comment:
Is this the max value allowed for both limits and requests?

Reply:
@4n4nd Yes, this will be for both limits and requests. `containerPolicies` has another field called `controlledValues` and default of this is `RequestsAndLimits`. So these apply to both of it. 

ContainerResourcePolicy ref: https://github.com/kubernetes/autoscaler/blob/master/vertical-pod-autoscaler/pkg/apis/autoscaling.k8s.io/v1/types.go#L153